index.html

```diff
@@ -1077,24 +1078,6 @@ <h4 class="pricing-col-header-amount"><span class="pricing-col-amount">$200</spa
   </div>
 </section>
 <!-- End Price Section -->
-<!-- Map Section -->
-<section id="map" class="dark">
-  <div id="canvas-map"></div>
-  <div class="container">
-    <div class="row">
-      <div class="col-md-5 col-sm-7 animated hiding location" data-animation="fadeInLeft" data-delay="0">
-        <h3><a href="location">Location</a></h3>
-        <ul itemscope="" itemtype="http://schema.org/PostalAddress">
-          <li itemprop="address"><i class="fa fa-map-marker highlight"></i>1500 Remount Rd, Front Royal, VA 22630</li>
-          <li itemprop="email"><i class="fa fa-envelope highlight"></i>info@rubyforgood.org</li>
-          <li itemscope="" itemtype="http://schema.org/Event"><i class="fa fa-clock-o highlight"></i>
-          <time itemprop="startDate" datetime="2016-06-16T07:30">3:00PM 6/16</time> - <time itemprop="endDate" datetime="2016-06-19T22:00">5:00PM 6/19</time>
-          </li>
-        </ul>
-      </div>
-    </div>
-  </div>
-</section>
```

Comment:
I think we want to keep this section on the homepage. It looks nice and does keep the address on the initial page.